src/main/java/de/qabel/desktop/storage/DirectoryMetadata.java

```diff
@@ -16,7 +16,7 @@
 import java.util.concurrent.Callable;
 
 public class DirectoryMetadata extends AbstractMetadata {
-
+    public static final long DEFAULT_SIZE = 56320L;
```

Comment:
As said before: This depends on the SQLite version and it can grow in large directories.